llvm-external-projects/iree-dialects/lib/Dialect/LinalgTransform/IR/LinalgTransformOps.cpp

```diff
@@ -204,7 +204,10 @@ FailureOr<LinalgOp> transform::FuseOp::applyToOne(LinalgOp target) {
 
 LogicalResult transform::FuseOp::verify() {
   SmallVector<int64_t> permutation = extractI64Array(tile_interchange());
-  auto sequence = llvm::seq<int64_t>(0, permutation.size());
```

Comment:
You could use
```
llvm::to_vector(llvm::seq<int64_t>(0, permutation.size());
```

Reply:
Ah! Much cleaner and still works, thanks.